spec/javascripts/mobile/mobile_comments_spec.js

```diff
@@ -82,14 +82,8 @@ describe("Diaspora.Mobile.Comments", function(){
   describe("createComment", function () {
     beforeEach(function() {
       spec.loadFixture("aspects_index_mobile_post_with_comments");
-      var commentBoxHtml = spec.fixtureHtml("comments_mobile_commentbox");
       var link = $(".stream .comment-action").first();
       Diaspora.Mobile.Comments.showCommentBox(link);
-      jasmine.Ajax.requests.mostRecent().respondWith({
```

Comment:
In this case, no ajax request should be performed during the test. So I'm actually not quite sure to which ajax call this line was supposed to answer.